Library/Formula/openlldp.rb

```diff
@@ -0,0 +1,19 @@
+class Openlldp < Formula
+  desc "Provides LLDP service and viewing from the command-line"
+  homepage "http://openlldp.sourceforge.net"
+  url "https://downloads.sourceforge.net/project/openlldp/openlldp-0.4alpha.tar.gz"
```

Comment:
Unless there's a stable release available here this will need to be filed in https://github.com/Homebrew/homebrew-devel-only.